src/full/Agda/Interaction/Options/Warnings.hs

```diff
@@ -58,19 +60,37 @@ defaultWarningMode :: WarningMode
 defaultWarningMode = WarningMode ws False where
   ws = fst $ fromMaybe __IMPOSSIBLE__ $ lookup defaultWarningSet warningSets
 
+-- | Some warnings are errors and cannot be turned off.
+data WarningModeError = Unknown String | NoNoError String
+
+prettyWarningModeError :: WarningModeError -> String
+prettyWarningModeError = \case
+  Unknown str -> concat [ "Unknown warning flag: ", str, "." ]
+  NoNoError str -> concat [ "Warnings for non-fatal errors such as "
```

Comment:
This wording sounds like if warnings for _fatal errors_ __could__ be turned off, which is probably not the intended connotation.